BackEndApp/authv1/emails.py

```diff
@@ -0,0 +1,28 @@
+from django.core.mail import send_mail
+from BackEndApp.settings import EMAIL_HOST_USER
+
+
+class Email_Templates:
+    def __init__(self, user, request):
+        self.user = user
+        self.request = request
+
+    def update_password(self, username, email, otp):
```

Comment:
```suggestion
    def forgot_password(self, username, email, otp):
```

The template is for forgot password right?